queries.sql

```diff
@@ -0,0 +1,27 @@
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT name FROM animals WHERE neutered = TRUE AND escape_attempts < 3;
+SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+SELECT name FROM animals WHERE neutered = TRUE;
+SELECT name FROM animals WHERE name!='Gabumon';
+SELECT name FROM animals WHERE weight_kg>=10.5 AND weight_kg <= 17.3;
+BEGIN;
```

Comment:
- [ ] You have done well writing out the queries but Looking at the requirement, you are missing the first 3 queries.
- - Screenshot reference:
![Selection_009](https://github.com/Nessrine88/Vet-clinic-database/assets/107632780/49a68c70-a5bc-4340-b445-cfc79a7fedb3)
- [ ] I'll suggest you go back to the requirement, reed the instructions, and write out these queries. :100: :1st_place_medal: 

Also, remember to update the screenshot accordingly :smile: :wave: